gateserver/controller_server.py

```diff
@@ -0,0 +1,17 @@
+"""The UDP server that provides the API for the controllers."""
+
+from . import controller_api
+import socketserver
+
+class MessageHandler(socketserver.BaseRequestHandler):
+    """Handles a message from the controller."""
+
+    def handle(self):
+        indata, socket = self.request
+        outdata = controller_api.handle_request(indata)
+        socket.sendto(outdata, self.client_address)
```

Comment:
Áno, je to `(ip, port)`. Inak by to nemohlo fungovať :D (Note: Nemám napísané testy, ale funguje to.)